ClassCastException: ZUGFeRDExporterFromA3 cannot be cast to ZUGFeRDExporterFromPDFA

When using
            `IZUGFeRDExporter ze = new ZUGFeRDExporterFromPDFA();`
then
`            ze.setProfile( Profiles.getByName( "extended" ) );`
leads to a class cast exception:
```Caused by: java.lang.ClassCastException: org.mustangproject.ZUGFeRD.ZUGFeRDExporterFromA3 cannot be cast to org.mustangproject.ZUGFeRD.ZUGFeRDExporterFromPDFA
	at org.mustangproject.ZUGFeRD.ZUGFeRDExporterFromPDFA.setProfile(ZUGFeRDExporterFromPDFA.java:151)
	at org.mustangproject.ZUGFeRD.ZUGFeRDExporterFromPDFA.setProfile(ZUGFeRDExporterFromPDFA.java:40)
	at biz.mbisoftware.fn.ejb.session.sales.InvoiceExport.buildInvoiceZugferdPdf(InvoiceExport.java:523)
	at biz.mbisoftware.fn.ejb.session.sales.InvoiceExport.processInvoice(InvoiceExport.java:403)
```

--- a/library/src/main/java/org/mustangproject/ZUGFeRD/ZUGFeRDExporterFromPDFA.java
+++ b/library/src/main/java/org/mustangproject/ZUGFeRD/ZUGFeRDExporterFromPDFA.java
@@ -147,16 +147,16 @@ public class ZUGFeRDExporterFromPDFA implements IZUGFeRDExporter {
 		return getExporter().setCreator(creator);
 	}
 
-	public ZUGFeRDExporterFromPDFA setProfile(Profile p) {
-		return (ZUGFeRDExporterFromPDFA) getExporter().setProfile(p);
+	public IZUGFeRDExporter setProfile(Profile p) {
+		return (IZUGFeRDExporter) getExporter().setProfile(p);
 	}
 
-	public ZUGFeRDExporterFromPDFA setProfile(String profileName) {
+	public IZUGFeRDExporter setProfile(String profileName) {
 		Profile p = Profiles.getByName(profileName);
 		if (p==null)  {
 			throw new RuntimeException("Profile not found.");
 		}
-		return (ZUGFeRDExporterFromPDFA) getExporter().setProfile(p);
+		return (IZUGFeRDExporter) getExporter().setProfile(p);
 	}
 
 	public IZUGFeRDExporter setConformanceLevel(PDFAConformanceLevel newLevel) {

